pcsx2/x86/ix86-32/iR5900-32.cpp

```diff
@@ -1315,17 +1315,29 @@ void recompileNextInstruction(int delayslot)
 			case 1:
 				switch(_Rt_) {
 					case 0: case 1: case 2: case 3: case 0x10: case 0x11: case 0x12: case 0x13:
-						Console.Warning("branch %x in delay slot!", cpuRegs.code);
+						DevCon.Warning("branch %x in delay slot!", cpuRegs.code);
 						_clearNeededX86regs();
 						_clearNeededXMMregs();
+						pc += 4;
+						g_cpuFlushedPC = false;
+						g_cpuFlushedCode = false;
+						if (g_maySignalException)
+							xAND(ptr32[&cpuRegs.CP0.n.Cause], ~(1 << 31)); // BD
```

Comment:
According to the documents, BD consists of two bits (BD & BD2) and both of them seem to have the same functionality, shouldn't you also clear the second bit if you're doing it for the first one? Any specific reason to only clear the first one?